src/GCPClientManager.php

```diff
@@ -164,4 +179,14 @@ public function close(): void
             $client->close();
         }
     }
+
+    private function setTimeouts(Google_Client $client): void
+    {
+        $guzzleClient = $client->getHttpClient();
+
+        $config = $guzzleClient->getConfig();
+        assert(is_array($config));
+        $config = array_merge($config, self::CONNECTION_TIMEOUTS_MAP);
```

Comment:
ale takto asi mozeme nastavovat tie timeouty co @zajca ?

Reply:
Díval sem sa na to, co to dělá, měl sem obavu, že by sa tam třeba nenastavily správně handlery nebo něco a vypadá to, že ne.